src/MessagingClient.ts

```diff
@@ -59,13 +59,15 @@ export class MessagingClient implements MessagingAPI {
                 conversation: {
                     id: room.roomId,
                     type: getConversationTypeFromRoom(this.matrixClient, room),
+                    lastEventTimestamp: room.timeline[room.timeline.length - 1].getTs(),
```

Comment:
Two questions here:
1- Could it happen that a timeline is empty?
2- Do we care about any last event or should we be filtering by `RoomMessage` as in the line below?

Reply:
On 1) It's technically possible, but shouldn't happen since if the conversation exists the `m.room.create` event should exist always.

2) This is to also have sorted by creation date in case there has never been a message sent

Reply:
I see, as we are already filtering by membership join I think we can rely on at least the creation being properly done. Thanks for clarifying!